app/assets/src/components/views/bulk_download/choose_step.scss

```diff
@@ -175,10 +175,41 @@
   }
 }
 
-.notification {
-  margin-bottom: $space-m;
+.highlight {
+  @include font-header-xxs;
+}
 
-  .filteredSamplesWarning {
-    @include font-header-xs;
+.listContainer {
+  background-color: $warning-bg;
+  margin-bottom: $space-s;
+}
+
+.notificationContainer {
+  @include font-body-xxs;
+  margin: $space-s 0;
+  background-color: $warning-bg;
+}
+
+.messageContainer {
+  max-height: 150px;
+  overflow: auto;
+  background-color: $warning-bg;
+  margin-left: 30px; // to align text with icon
```

Comment:
It's a little weird to have this value depend on a css value in another component. You could solve this by moving the rendering of the list of strings into compact list notification as I describe above.